typescript/src/http_controller/itask_http_controller.ts

```diff
@@ -0,0 +1,58 @@
+import {HttpRequestWithIdentity} from '@essential-projects/http_contracts';
+
+import {Response} from 'express';
+
+/**
+ * Describes a HTTPController for managing HttpRequests related to Tasks.
+ */
+export interface ITaskHttpController {
```

Comment:
Den Controller würde ich dann auch so nennen:

```suggestion
export interface IFlowNodeInstanceHttpController {
```